README.md

```diff
@@ -1,6 +1,6 @@
-## SuiteCRM 7.9.1
+## SuiteCRM 7.9.2
```

Comment:
sorry, newbie here... I only included the changed file... do you want me to include only the changed lines? not sure about how to do that...

and what do I have to remove?